contentcuration/contentcuration/frontend/channelEdit/components/edit/CompletionOptions.vue

```diff
@@ -324,6 +323,13 @@
                 model: CompletionCriteriaModels.REFERENCE,
                 threshold: null,
               };
+            } else if (value === CompletionDropdownMap.completeDuration) {
+              // set to '1' as the minimum here, due to validation rules requiring > 0
```

Comment:
Not sure I understand why we're setting a default here - feels like if there is no value, we need to let the user set it.